lib/wallaroo/core/sink/tcp_sink/tcp_sink_builder.pony

```diff
@@ -115,12 +115,16 @@ class val TCPSinkBuilder
   fun apply(sink_name: String, event_log: EventLog,
     reporter: MetricsReporter iso, env: Env,
     barrier_coordinator: BarrierCoordinator, checkpoint_initiator: CheckpointInitiator,
-    recovering: Bool, worker_name_: WorkerName, auth_: AmbientAuth): Sink
+    recovering: Bool, app_name: String, worker_name: WorkerName,
+    auth_: AmbientAuth): Sink
```

Comment:
same points regarding naming

Reply:
I'm trying to indicate that the variable is unused.

Reply:
The `SinkBuilder` interface requires the last `AmbientAuth` arg. `ConnectorSink` needs it, but `TCPSink` does not.  I'll remove the trailing underscore.